pyvista/examples/downloads.py

```diff
@@ -4933,6 +4933,48 @@ def download_dikhololo_night():  # pragma: no cover
     return texture
 
 
+def download_hydrogen_orbital(load=True):  # pragma: no cover
```

Comment:
This is definitely the physicist talking in me, but can we squeeze in "3d orbital" somewhere? The ground state of hydrogen has sphericslly symmetric density. The 3d orbital is a higher-energy excited state. (You actually see similar electron orbitals on transition-metal atoms such as iron).

Reply:
This is why we have different reviewers. My understanding of orbitals is limited to high school physics.

Fixing.

Reply:
```suggestion
def download_hydrogen_orbital_3d(load=True):  # pragma: no cover
```

Reply:
Sorry, I think I misplaced my comment. I only meant the docstring header line. The function name is fine until we ever add other orbitals.